parser.y

```diff
@@ -2937,7 +2939,7 @@ IndexOption:
 	{
 		$$ = &ast.IndexOption{
 			// TODO bug should be fix here!
```

Comment:
PTAL @XuHuaiyu 
what should the bug fix here?
Do we support `KeyBlockSize` function?